src/components/Strip/Strip.tsx

```diff
@@ -1,26 +1,44 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
 import React from "react";
+import type { ElementType, HTMLProps, ReactNode } from "react";
 
 import Col, { colSizes } from "../Col";
+import type { ColSize } from "../Col/Col";
 import Row from "../Row";
 
+export type Props = {
+  children: ReactNode;
+  background?: string;
+  bordered?: boolean;
+  className?: string;
+  colSize?: ColSize;
+  dark?: boolean;
+  deep?: boolean;
+  element?: ElementType;
+  includeCol?: boolean;
+  light?: boolean;
+  rowClassName?: string;
+  shallow?: boolean;
+  type?: string;
+} & HTMLProps<HTMLElement>;
```

Comment:
Forgot about this in the other PRs but we should add descriptions for each prop

Reply:
Yeah good idea, might have to go back over the other components. Also might be worth trying to generate the prop types once they're all migrated: https://github.com/canonical-web-and-design/react-components/issues/201.

Reply:
I've pushed these.